### PR TITLE
Update Multimodal Docker File Path

### PR DESCRIPTION
**Pull request by letonghan**
## Description

Update Multimodal Docker File Path

## Issues

Fix CI issue of [this pr](https://github.com/opea-project/GenAIComps/pull/1016).

## Type of change

List the type of change like below. Please delete options that are not relevant.

- [x] Others (enhancement, documentation, validation, etc.)

## Dependencies

None

## Tests

Local tested

### CODE REVIEW

**Review by ZePan110 — APPROVED** [review]

File: MultimodalQnA/docker_compose/intel/cpu/xeon/README.md [hunk 1]
```diff
@@ -124,7 +124,7 @@ docker build --no-cache -t opea/embedding-multimodal:latest --build-arg https_pr
 ### 2. Build retriever-multimodal-redis Image
 
 ```bash
-docker build --no-cache -t opea/retriever-multimodal-redis:latest --build-arg https_proxy=$https_proxy --build-arg http_proxy=$http_proxy -f comps/retrievers/multimodal/redis/langchain/Dockerfile .
+docker build --no-cache -t opea/retriever-multimodal-redis:latest --build-arg https_proxy=$https_proxy --build-arg http_proxy=$http_proxy -f comps/retrievers/redis/langchain/Dockerfile .
```
Comment (chensuyue) [inline MultimodalQnA/docker_compose/intel/cpu/xeon/README.md:127]: Do we still need this `retriever-multimodal-redis` image?

Comment (chensuyue) [reply MultimodalQnA/docker_compose/intel/cpu/xeon/README.md:127]: This file also need to update https://github.com/opea-project/GenAIExamples/blob/da8c92e7c5c855b660a44569012dee71b513770c/MultimodalQnA/docker_image_build/build.yaml#L32

Comment (chensuyue) [reply MultimodalQnA/docker_compose/intel/cpu/xeon/README.md:127]: Also check this one, https://github.com/opea-project/GenAIExamples/blob/da8c92e7c5c855b660a44569012dee71b513770c/MultimodalQnA/docker_compose/intel/hpu/gaudi/compose.yaml#L77

Comment (letonghan) [reply MultimodalQnA/docker_compose/intel/cpu/xeon/README.md:127]: The image name should be unified to `retriever-redis`. Will fix this.

**Comment by letonghan** [comment]
The CI test on xeon and gaudi will fail because codes in GenAIComps are not merged yet. Need to merge [comps pr](https://github.com/opea-project/GenAIComps/pull/1016) first, then trigger CI here again. @chensuyue

**Review by lvliang-intel — APPROVED** [review]

**Pull request merged**